pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/StudyGovernanceDao.sql.stg

```diff
@@ -36,3 +41,35 @@ queryPolicyByStudyGuid() ::= <<
 where study.guid = :studyGuid
 order by aom.execution_order asc
 >>
+
+select_all_age_up_candidates() ::= <<
+select auc.age_up_candidate_id,
+       study.umbrella_study_id as study_id,
+       study.guid              as study_guid,
+       participant.user_id as participant_user_id,
+       participant.guid    as participant_user_guid,
+       profile.birth_date,
+       auc.initiated_preparation,
+       (select et.enrollment_status_type_code
```

Comment:
Might be worth adding some fail-safe logic in here to assure a single row is returned.  The schema doesn't have a unique constraint that involves `valid_to`.  Maybe order `desc` by `last_updated` and pick off the first row?  Since these are candidates, further filtering will take place, so even if we have too many candidates, there's another layer of defense.

Reply:
Added `order by valid_from desc limit 1`.